.eslintrc

```diff
@@ -27,7 +27,8 @@
         "singleQuote": true,
         "tabWidth": 2,
         "useTabs": false,
-        "trailingComma": "all"
+        "trailingComma": "all",
+        "arrowParens": "avoid",
```

Comment:
Prettier 2.0 changed this rule: https://prettier.io/docs/en/options.html#arrow-function-parentheses

This puts it back to the 1.0 setting so we don't have to change any of the parens right now.